### PR TITLE
fix: Use snackbar instead of Toast

### LINKED ISSUE #8: Use snackbar instead of Toast
**Describe the feature you'd like**

Replace the Toast with snackbar.

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by aman-g0yaa**
Fixes #8 

**Changes:**

 Replaced the Toast with snackbar.

- [x] Run unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into a single commit by squashing them.

 ### Screenshots:
<table>
  <tr>
    <td></td>
    <td></td>
      </tr>
  <tr>

### CODE REVIEW

**Comment by immadisairaj** [comment]
@amangoyal2339 please follow semantic pr guidelines

**Comment by aman-g0yaa** [comment]
@immadisairaj  please Review .

**Comment by immadisairaj** [comment]
> @immadisairaj please Review .

yeah, sure. No need to ask me to review. I'll do it sometime or other.

File: app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java [hunk 1]
```diff
@@ -85,14 +85,17 @@ public void onResponse(Call<Info> call, Response<Info> response) {
                         swipeContainer.setRefreshing(false);
                         View loadingIndicator = findViewById(R.id.loading_indicator_info);
                         loadingIndicator.setVisibility(View.INVISIBLE);
-                        Toast.makeText(getApplicationContext(), handle, Toast.LENGTH_SHORT).show();
+                        Snackbar.make(getWindow().getDecorView().getRootView(), handle, Snackbar.LENGTH_SHORT)
+                                .show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java]: can you move this to the above line

File: app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java [hunk 2]
```diff
@@ -85,14 +85,17 @@ public void onResponse(Call<Info> call, Response<Info> response) {
...
                         showInfo(info);
                     } else {
-                        Toast.makeText(getApplicationContext(), "Wrong handle, Try Again", Toast.LENGTH_SHORT).show();
+                        Snackbar.make(getWindow().getDecorView().getRootView(), "Wrong handle, Try Again", Snackbar.LENGTH_SHORT)
+                                .show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java]: same as above

**Review by immadisairaj — CHANGES_REQUESTED** [review]
can you make all the statements in a single line

File: app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java [hunk 3]
```diff
@@ -85,25 +85,25 @@ public void onResponse(Call<Info> call, Response<Info> response) {
                         swipeContainer.setRefreshing(false);
                         View loadingIndicator = findViewById(R.id.loading_indicator_info);
                         loadingIndicator.setVisibility(View.INVISIBLE);
-                        Toast.makeText(getApplicationContext(), handle, Toast.LENGTH_SHORT).show();
+                        Snackbar.make(getWindow().getDecorView().getRootView(), handle, Snackbar.LENGTH_SHORT) .show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java]: remove the space near `RT) .show(`

File: app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java [hunk 4]
```diff
@@ -85,25 +85,25 @@ public void onResponse(Call<Info> call, Response<Info> response) {
...
 
             }
 
             @Override
             public void onFailure(Call<Info> call, Throwable t) {
                 swipeContainer.setRefreshing(false);
-                Toast.makeText(getApplicationContext(), "No Internet Connection", Toast.LENGTH_SHORT).show();
-                if (countOfCalls == 0)
-                    InfoActivity.super.onBackPressed();
+                Snackbar.make(getWindow().getDecorView().getRootView(), "No Internet Connection", Snackbar.LENGTH_SHORT)  .show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java]: same here

File: app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java [hunk 5]
```diff
@@ -85,25 +85,25 @@ public void onResponse(Call<Info> call, Response<Info> response) {
...
+                  if (countOfCalls == 0)
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java]: can you remove the leading extra space

File: app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java [hunk 6]
```diff
@@ -85,25 +85,25 @@ public void onResponse(Call<Info> call, Response<Info> response) {
...
+                    InfoActivity.super.onBackPressed();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/InfoActivity.java]: can you remove the trailing spaces

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java [hunk 7]
```diff
@@ -96,23 +96,24 @@ public void onResponse(Call<Submission> call, Response<Submission> response) {
...
-                        Toast.makeText(getApplicationContext(), handle, Toast.LENGTH_SHORT).show();
-                        showSubmissions(submission);
+                        Snackbar.make(getWindow().getDecorView().getRootView(),  handle, Snackbar.LENGTH_SHORT) .show();
+                            showSubmissions(submission);    
+                       
                     } else {
-                        Toast.makeText(getApplicationContext(), "Wrong handle, Try Again", Toast.LENGTH_SHORT).show();
-                        SubmissionActivity.super.onBackPressed();
+                        Snackbar.make(getWindow().getDecorView().getRootView(),  "Wrong handle, Try Again", Snackbar.LENGTH_SHORT).show();
+                            SubmissionActivity.super.onBackPressed();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java]: can you please remove the extra leading and trailing spaces

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java [hunk 8]
```diff
@@ -96,23 +96,24 @@ public void onResponse(Call<Submission> call, Response<Submission> response) {
...
                     } else {
-                        Toast.makeText(getApplicationContext(), "Wrong handle, Try Again", Toast.LENGTH_SHORT).show();
-                        SubmissionActivity.super.onBackPressed();
+                        Snackbar.make(getWindow().getDecorView().getRootView(),  "Wrong handle, Try Again", Snackbar.LENGTH_SHORT).show();
+                            SubmissionActivity.super.onBackPressed();   
                     }
                 } else {
-                    Toast.makeText(getApplicationContext(), "Wrong handle, Try Again", Toast.LENGTH_SHORT).show();
-                    SubmissionActivity.super.onBackPressed();
+                    Snackbar.make(getWindow().getDecorView().getRootView(),  "Wrong handle, Try Again", Snackbar.LENGTH_SHORT) .show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java]: same as above

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java [hunk 9]
```diff
@@ -96,23 +96,24 @@ public void onResponse(Call<Submission> call, Response<Submission> response) {
...
+                     SubmissionActivity.super.onBackPressed();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java]: same as above

File: app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java [hunk 10]
```diff
@@ -96,23 +96,24 @@ public void onResponse(Call<Submission> call, Response<Submission> response) {
...
+                     SubmissionActivity.super.onBackPressed();     
                 }
 
             }
 
             @Override
             public void onFailure(Call<Submission> call, Throwable t) {
                 swipeContainer.setRefreshing(false);
-                Toast.makeText(getApplicationContext(), "No Internet Connection", Toast.LENGTH_SHORT).show();
+                Snackbar.make(getWindow().getDecorView().getRootView(),  "No Internet Connection", Snackbar.LENGTH_SHORT) .show();
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java]: same as above

**Review by immadisairaj — CHANGES_REQUESTED** [review]
please try to change it in local but not directly using the github. and also squash the commits into one

**Comment by immadisairaj** [comment]
@amangoyal2339 why are they multiple commits. and which were already there..

can you resolve the conflicts

**Pull request closed without merging**